content/automate/ManageIQ/AutomationManagement/AnsibleTower/Operations/StateMachines/Job.class/__methods__/launch_ansible_job.rb

```diff
@@ -116,7 +115,7 @@ def run(job_template, target)
                   args[:limit] = target if target
                   @handle.log(:info, "Job Arguments #{args}")
 
-                  job = @handle.vmdb(JOB_CLASS).create_job(job_template, args)
+                  job = job_template.create_job(args)
```

Comment:
@lfu will they cause any backward compatibility issues for users who have copied this method into their own domain. Will the old code still work

Reply:
The old method to create a job as a class method still works. 
This PR allows to create a job as a template instance method.